pypeit/images/calibrationimage.py

```diff
@@ -7,71 +7,58 @@
 
 
 from pypeit import msgs
-
-from pypeit.core import combine
 from pypeit.par import pypeitpar
-
-from pypeit.images import pypeitimage
-from pypeit.images import processrawimage
-from pypeit.images import maskimage
-
-
+from pypeit.images import buildimage
 
 from IPython import embed
 
 
-class CalibrationImage(pypeitimage.PypeItImage, maskimage.ImageMask):
+class CalibrationImage(object):
```

Comment:
Because some of the `CalibrationImage` classes (e.g. `ArcImage`) have evolved to becoming the objects which generate a `PypeItImage`, while others generate entirely different things or mutliple `PypeItImage` objects (e.g. `FlatField`).  

I nearly eliminated this `CalibrationImage` object but decided that I didn't want to duplicate the `build_image()` method in the several places it is used.  And the file checking methods are somewhat useful.